### ISSUE TITLE
Discuss a Voicing Design Pattern for Radio Groups

### ISSUE THREAD

**Issue by terracoda** [phetsims/membrane-transport#146]
It's tricky to incorporate all the useful information a radio group needs for the Voicing feature.
The screen reader experience with Interactive Description automatically handles the group name and the count of the buttons.

The group name and count may seem like extra unneeded information for visual users of the Voicing feature.

So far in my Voicing design journey, I have come across 3 radio button groups:
1. "Tick Mark Options" in Ratio and Proportion - no text on screen
2. "Trig Functions" in Trig Tour - radio button names on screen
3. "Solutes" in Membrane Transport - group name "Solutes" on screen and part of the button name on screen.
 
> - The visible "Solutes" heading should be a mouse-only reading block. Keyboard users should get the group name on focus. 
> - I think I might need to create a Voicing Design Pattern for radio button groups. They are very tricky. In RaP I incorporated the group into the button name, and in Trig Tour we incorporated the group name into the help text. In Membrane Transport, the group name is on the screen, and I feel it could be voiced up focusing the group in the same way we are voicing help text upon entering the group.  I think this would be a good pattern in general for radio groups with 3 or more options, and could save design time. 
> 
> **Potential pieces of information are:**
> - voicingNameResponse 
> - radioButtonCount + totalRadioButtons
> - groupName -  autopupulated by the heading in the PDOM
> - voicingHelpText-IfChecked
> 
> **Ideal Information organization:**
> - have voicingGroupName and button count be an object response, so it can be turned off
> 
> ** Potential Concatenated Response upon enter a radio button group with keyboard:**
>     - voicingNameResponse + voicingObjectResponse(count + groupName) + voicingHelpText
> 
> For the Solutes radio buttons, it would be like this for keyboard interaction:
> - **On focus:** Oxygen, O2, nonpolar, small, 1 of 6 Solutes, Choose solute for outside or inside area of cell membrane.
> - On change: Carbon Dioxide, CO2, nonpolar, small, 2 of 6 Solutes
> 
> We can discuss.  

 _Originally posted by @terracoda in [#79](https://github.com/phetsims/membrane-transport/issues/79#issuecomment-2845437693)_

**Comment by jessegreenberg** [comment]
@terracoda @samreid and @jessegreenberg met today to discuss this. We created a common code issue with design ideas: https://github.com/phetsims/sun/issues/937.

**Comment by terracoda** [comment]
Work in Membrane Transport is complete. Common code and documentation improvements will continue in sun.
